docs/reference/tab-widgets/troubleshooting/data/diagnose-unassigned-shards.asciidoc

```diff
@@ -0,0 +1,287 @@
+////
+[source,console]
+----
+PUT my-index-000001?master_timeout=1s&timeout=1s
+{
+  "settings": {
+    "index.routing.allocation.include._name": "nonexistent_node",
+    "index.routing.allocation.include._tier_preference": null
+  }
+}
+----
+////
+
+// tag::cloud[]
+In order to diagnose the unassigned shards, follow the next steps:
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support],
+or your deployment doesn't include {kib}, in which case all you need to do is 
+{cloud}/ec-access-kibana.html[enable Kibana first].
+
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Dev Tools > Console**.
++
+[role="screenshot"]
+image::images/kibana-console.png[{kib} Console,align="center"]
```

Comment:
I feel like I've seen this sentence enough that we should make it an expandable token, like we do with `{es}` or `{kib}`

Reply:
Would it be ok if we do this in a follow-up? (I have no idea how to do this and it'll be easy enough to, once done, replace in these guides)

Reply:
Oh totally, not a blocker for this at all